daprdocs/content/en/developing-applications/integrations/AWS/authenticating-aws.md

```diff
@@ -10,22 +10,65 @@ aliases:
-| `endpoint` | The endpoint is normally handled internally by the AWS SDK. However, in some situations it might make sense to set it locally - for example if developing against [DynamoDB Local](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html). |
-| `accessKey` | AWS Access key id. |
-| `secretKey` | AWS Secret access key. Use together with `accessKey` to explicitly specify credentials. |
-| `sessionToken` | AWS Session token. Used together with `accessKey` and `secretKey`. When using a regular IAM user's access key and secret, a session token is normally not required. |
+### AWS: Access Key ID and Secret Access Key
+Authenticate to AWS using static Access Key and Secret Key information. You can do this through metadata fields on the component, or by using the [default AWS configuration](https://docs.aws.amazon.com/sdkref/latest/guide/creds-config-files.html) for this authentication profile. 
 
 {{% alert title="Important" color="warning" %}}
-You **must not** provide AWS access-key, secret-key, and tokens in the definition of the component spec you're using:
+You should leverage the loading of the default AWS configuration instead of providing AWS access key, secret key, and tokens in the definition of the component spec you're using:
 - When running the Dapr sidecar (`daprd`) with your application on EKS (AWS Kubernetes)
 - If using a node/pod that has already been attached to an IAM policy defining access to AWS resources 
 {{% /alert %}}
 
+| Attribute | Required | Description | Example |
+| --------- | ----------- | ----------- | ----------- |
+| `region` | Y | Which AWS region to connect to. | "us-east-1" |
+| `accessKey` | N | AWS Access key id. | "AKIAIOSFODNN7EXAMPLE" |
+| `secretKey` | N | AWS Secret access key. Use together with `accessKey` to explicitly specify credentials. | "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY" |
+| `sessionToken` | N | AWS Session token. Used together with `accessKey` and `secretKey`. When using a regular IAM user's access key and secret, a session token is normally not required. | |
+
+### Assume IAM Role
+Assume a specific IAM Role. This authentication profile assumes your running the Dapr sidecar on EKS (AWS Kubernetes), or using a node/pod that has already been attached to an IAM policy defining access to AWS resources.
+
+This authentication profile is only supported on Kafka and PostgreSQL components at this time.
+
+| Attribute | Required | Description | Example |
+| --------- | ----------- | ----------- | ----------- |
+| `region` | Y | Which AWS region to connect to. | "us-east-1" |
+| `assumeRoleArn` | N | IAM role that has access to AWS resource. This field will be marked required in Dapr 1.17. | "arn:aws:iam::123456789:role/mskRole" |
```

Comment:
should we say 'in a future release' instead of promising Dapr 1.17?

Reply:
for deprecating things it is proper to deprecate after two releases is why I put 1.17.